### ISSUE TITLE
How to Use an Image in a Basic HTML Page: A Step-by-Step Guide

### ISSUE THREAD

**Issue by yunusatalay** [yunusatalay/HTML#4]
In this article, we will discuss how to use an image in a basic HTML page. Images play an essential role in web development, as they can help to make a website more visually appealing and engaging. HTML provides a straightforward way to add images to a webpage. We will discuss the steps involved in adding an image to a webpage, including selecting an image, adding the image to the HTML code, and optimizing the image for web use. By the end of this article, you will be able to add images to your HTML pages with ease.

### **Step 1: Selecting an Image:**

The first step in adding an image to a webpage is selecting the image you want to use. There are various sources to get images for your website, such as stock photo sites, image search engines, or your own photos. When selecting an image, you should consider the following:

**Image Quality:** You should choose an image that is of high quality and resolution, as low-quality images can appear blurry or pixelated on some devices.
**Image Size:** The size of the image is also an essential consideration. Larger images take longer to load, which can slow down your website. Therefore, you should choose an image that is optimized for web use and does not exceed the required dimensions.
**Image Type:** The type of image file you choose is also important. Common file types used for web images are JPEG, PNG, and GIF.

### **Step 2: Adding the Image to the HTML Code:**

Once you have selected the image you want to use, the next step is to add it to your HTML code. You can add an image to your HTML code using the "img" tag. The "img" tag has the following syntax:

`<img src="image_file_path" alt="alternate_text">`

The "src" attribute specifies the file path of the image file, and the "alt" attribute specifies alternate text for the image. The alternate text is displayed if the image cannot be loaded or for users who use screen readers.

Here is an example of how to add an image to your HTML code:

`<img src="image.jpg" alt="A beautiful sunset">`

### **Step 3: Optimizing the Image for Web Use:**

Optimizing images for web use is important because it can improve your website's load time, which can improve user experience. Here are some tips for optimizing images for web use:

**Resize the Image:** You should resize the image to the required dimensions before uploading it to your website. Large images take longer to load, so you should resize them to the appropriate size to reduce load time.
**Compress the Image:** Compressing the image can reduce its file size without significantly reducing its quality. You can use various image compression tools available online to compress your images before uploading them to your website.
**Choose the Right Image Type:** Choosing the right image type can also help to optimize the image for web use. For example, JPEG images are best for photographs, while PNG images are best for graphics with transparent backgrounds.

**Conclusion:**
In conclusion, adding images to a webpage is an essential part of web development, and HTML provides an easy way to do so. You can add images to your HTML code using the "img" tag, and you should optimize the images for web use to improve load time and user experience. With the steps outlined in this article, you can now add images to your HTML pages with ease.
